spec/classes/lldpd_spec.rb

```diff
@@ -47,11 +47,15 @@
           it { is_expected.not_to contain_apt__source('lldpd') }
         end
 
-        if %w[CentOS RedHat].include? facts[:os]['family']
+        # rubocop:disable RSpec/RepeatedExample
```

Comment:
I'd prefer to avoid this. How about
```suggestion
        if %w[CentOS RedHat].include?(facts[:os]['family') && facts[:os]['name'] != 'VirtuozzoLinux'
```